### DISCUSSION TITLE
Proposal for a website, it can be hosted at github.io

### DISCUSSION THREAD

**Discussion by eznix86** [founded-labs/react-native-reusables#17] [Ideas]
Basically the website will be the basis of the project, there could be screenshot or gif of each components. We can use docsaurus for that, similar to React Native Paper.

Example:
https://callstack.github.io/react-native-paper/docs/components/ActivityIndicator

Later we can migrate to a custom website, like Astro using Shadcn UI with gif ;)

**Comment by mrzachnugent** [comment]
Started creating the docs: https://rnr-docs.vercel.app/
